pkgs/development/haskell-modules/configuration-ghc-8.0.x.nix

```diff
@@ -74,4 +74,82 @@ self: super: {
   # https://github.com/haskell-suite/haskell-src-exts/issues/302
   haskell-src-exts = dontCheck super.haskell-src-exts;
 
+  active         = doJailbreak super.active;
+
+  authenticate-oauth = doJailbreak super.authenticate-oauth;
+
+  diagrams-core  = doJailbreak super.diagrams-core;
+
+  diagrams-lib   = doJailbreak super.diagrams-lib;
+
+  foldl          = doJailbreak super.foldl;
+
+  force-layout   = doJailbreak super.force-layout;
+
+  # Partial fixes released in 1.20.5 upstream, full fixes only in git
+  linear         = pkgs.haskell.lib.overrideCabal super.linear (oldAttrs: {
+    editedCabalFile = null;
+    revision        = null;
+    src = pkgs.fetchgit {
+            url    = https://github.com/ekmett/linear.git;
+            rev    = "8da21dc72714441cb34d6eabd6c224819787365c";
+            sha256 = "08l0z6zrlbals2bwc2abbh31j9gf90vgp8sy3dcrp0knc98bgaa1";
+    };
+  });
+
+  lucid-svg      = doJailbreak super.lucid-svg;
+
+  monads-tf      = doJailbreak super.monads-tf;
+
+  parsers        = doJailbreak super.parsers;
+
+  pointed        = super.pointed_5;
+
+  reducers       = doJailbreak super.reducers;
+
+  sdl2           = doJailbreak super.sdl2;
+
+  servant        = dontCheck (doJailbreak super.servant_0_7);
+  servant-client = dontCheck (doJailbreak super.servant-client_0_7);
+  servant-server = dontCheck (doJailbreak super.servant-server_0_7);
+
+  # The essential part is released in 2.1 upstream (needs hackage import)
+  singletons     = (pkgs.haskell.lib.overrideCabal super.singletons (oldAttrs: {
```

Comment:
Version 2.1? I don't see that on http://hackage.haskell.org/package/singletons. The latest one is 2.0.1, which we have. Am I missing something?